src/EventTypeDefs.ts

```diff
@@ -129,9 +129,21 @@ export interface TPointerEventInfo<E extends TPointerEvent = TPointerEvent>
   extends TEvent<E> {
   target?: FabricObject;
   subTargets?: FabricObject[];
-  pointer: Point;
   transform?: Transform | null;
+  /**
+   * @deprecated
+   * use viewportPoint instead.
+   * Kept for compatibility
+   */
+  pointer: Point;
+  /**
+   * @deprecated
+   * use scenePoint instead.
+   * Kept for compatibility
+   */
   absolutePointer: Point;
+  scenePoint: Point;
+  viewportPoint: Point;
```

Comment:
the order is confusing 


should have been 

```
pointer,
absolutePointer,
viewportPoint,
scenePoint
```

OR

```
pointer,
viewportPoint,
absolutePointer,
scenePoint
```


It might seem petty or insignificant but IMO it isn't
A dev reading this code will scratch their hand